resources/definitions/ultimaker.def.json

```diff
@@ -82,6 +82,67 @@
         },
         "xy_offset": {
             "value": "-layer_height * 0.2"
+        },
+        "bridge_settings_enabled": {
+            "value": false
```

Comment:
But that's the default, right?

Reply:
Yep, for this pull request we decided to keep the bridge settings off. This improves the settings when people enable it.
We have a ticket open to get bridge settings out of experimental. When this happens we will enable it by default in our profiles.